libcodechecker/libhandlers/parse.py

```diff
@@ -185,13 +193,23 @@ def main(args):
         sys.exit(2)
 
     for input_path in args.input:
+
         input_path = os.path.abspath(input_path)
         os.chdir(original_cwd)
         LOG.debug("Parsing input argument: '" + input_path + "'")
 
+        severity_stats = Counter({})
+        file_stats = Counter({})
+
         if os.path.isfile(input_path):
-            parse(input_path, context, {}, suppress_handler,
-                  'print_steps' in args)
+            report_stats = parse(input_path,
```

Comment:
There is a lot of code duplication here. If the input path is a file we could create a one element list containing that file and fall back to the code bellow avoid duplication.

Reply:
Done.